cli/src/command/account.rs

```diff
@@ -382,9 +383,9 @@ pub async fn claimable_outputs_command(account: &Account) -> Result<(), Error> {
             println_log_info!("  - base coin amount: {}", amount);
 
             if let Some(expiration) = unlock_conditions.expiration() {
-                let current_time = iota_sdk::utils::unix_timestamp_now().as_secs() as u32;
-                let time_left = expiration.timestamp() - current_time;
-                println_log_info!("  - expires in: {} seconds", time_left);
+                let slot_index = account.client().get_slot_index().await?;
+                let slot_indices_left = *expiration.slot_index() - *slot_index;
```

Comment:
It could already overflow :trollface:

Reply:
I can print an alternative if expired

Reply:
> It could already overflow :trollface:

Yes, doesn't make it better :trollface: Print alternative or maybe just convert to an i128 so it can be negative?

Reply:
Added an alternative print